dd-sdk-android/src/main/kotlin/com/datadog/android/error/internal/DatadogExceptionHandler.kt

```diff
@@ -43,6 +52,19 @@ internal class DatadogExceptionHandler(
         )
         writer.write(log)
 
+        contextRef.get()?.let {
+            println("Trigger work manager task")
```

Comment:
Maybe we should create a task to schedule this Worker when the SDK is initialised to actually try to send the logs whenever the device comes from an offline state into an online one if this is possible. I am thinking that this can be useful for the moments the user used the app in offline, closed the app and then it gets network connection. This will assure us that the logs stored in offline will be actually send.

Reply:
That's an idea, but not in the scope of this PR, I created a JIRA task for it (RUMM-169).
I'm not that convinced it should be triggered at the start of the app though, cause the handler is gonna loop anyway, but when the app stops, that might be something interesting.